pkg/gen/nodejs-templates/kind.ts.mustache

```diff
@@ -46,7 +46,7 @@ import { getVersion } from "../../version";
 
       /**
        * Create a {{Group}}.{{Version}}.{{Kind}} resource with the given unique name, arguments, and options.
-       *
+       {{{AwaitComment}}}
```

Comment:
I was imagining we would put this on the class - not the constructor. I think it would be a little unexpected on the constructor.